Create movable-ink-event-streaming.md (#913)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/destinations/movable-ink-event-streaming.md

---
title: Movable Ink Event Streaming
description: Stream Amplitude events to Movable Ink.
status: new
---

!!!beta "This feature is in open beta"

    This feature is in open beta and is in active development. Contact the [Movable Ink Client Experience team](https://movableink.com/contact-us) for support with this integration.

[Movable Ink](https:///movableink.com) activates any data into personalized content in any customer engagement and is a powerful extension to the technologies that marketers use today. The platform easily connects to all relevant data—no matter where it lives—updates based upon a recipient's most recent interactions, and is auto-generated by the platform.

## Considerations

- You must enable this integration in each Amplitude project you want to use it in.

> - You need to be an admin of a paid Planhat plan to enable this integration.

Author: Thanks for catching this.

- Relevant limits for Movable Ink events are:
  - Movable Ink does not impose any hard limits on quantity or velocity; however, requests sent too quickly will return 429 responses. Amplitude will handle these automatically. 
  - The requests must be smaller than 2MB.
- Amplitude sends select event and user properties along with the event.

## Setup

### Prerequisites

> To configure an Event Streaming integration from Amplitude to Movable Ink, you must fulfill the following prerequisites from Movable Ink:

Author: Made the changes.

2. Enter a sync name, then click **Create Sync**.
3. Toggle Status from **Disabled** to **Enabled**.
4. Paste your **Endpoint URL (taken from the Movable Ink platform)**.
5. Paste in your **Basic Authorization Header Value**.
6. Toggle the **Send events**.
7. In **Select and filter events** choose which events you want to send. Choose only the events you need in Movable Ink. Transformed events aren't supported.
8. (optional) In **Select additional properties**, select any more event and user properties you want to send to Movable Ink. If you don't select any properties here, Amplitude doesn't send any. Transformed event properties and transformed user properties aren't supported.
9. Under **Send Users**, make sure the toggle is enabled ("Users are sent to MovableInk") if you want to stream users and their properties to MovableInk. When enabled, users are automatically created or updated in MovableInk when an event is sent to Amplitude. [Amplitude Identify API](https://www.docs.developers.amplitude.com/analytics/apis/identify-api/) calls are also forwarded to MovableInk. Users aren't sent on a schedule or on-demand using this integration.

> 10. (optional) In **Select additional properties**, select any more user properties you want to send to MovableInk. If you don't select any properties here, Amplitude doesn't send any. _Transformed user properties aren't supported._

Reviewer: ```suggestion
10. (optional) In the **Select additional properties** section, select any additional user properties you want to send to Movable Ink. Amplitude sends only the properties you select here. This integration does not support transformed user properties. 
```

11. When finished, enable the destination and **Save**.

_Once the initial endpoint setup is finalized within Movable Ink and Amplitude, event data will need to be mapped within Movable Ink. Work with your Movable Ink CXM on next-steps._

### Use Cases

- **Generate personalized content:** Generate personalized content relevant to each user using event data from Amplitude.

> - **Analyzing and Segmenting Users:** With Amplitude's analytics capabilities, you can analyze user behavior, identify patterns, and create segments based on specific actions or attributes. For example, you may identify a segment of users who have recently made a purchase or visited a specific page.

Reviewer: ```suggestion
- **Analyze and Segment Users:** With Amplitude's analytics capabilities, you can analyze user behavior, identify patterns, and create segments based on specific actions or attributes. For example, you may identify a segment of users who have recently made a purchase or visited a specific page.
```

> - **Sending Event Data to Movable Ink:** Once you have defined your desired segments, you can set up an integration or data pipeline to send the relevant event data from Amplitude to Movable Ink. This could be achieved through APIs, webhook integrations, or other methods supported by both platforms.

Reviewer: ```suggestion
- **Send Event Data to Movable Ink:** After you define your segments, you can configure an integration or data pipeline to send the relevant event data from Amplitude to Movable Ink. You can do this with APIs, web hook integrations, or other methods supported by both platforms.
```
